tasks/zaproxy.js

```diff
@@ -50,9 +52,17 @@ module.exports = function (grunt) {
 
     // Spawn ZAProxy
     // var zapPath = path.join(__dirname, '../vendor/zap');
-    // var cmd = path.join(options.path, 'zap.sh');
+
     grunt.log.write('Starting ZAProxy: ');
-    var child = spawn('zap.sh', args);
+
+    var zapScript = (options.os === 'windows' ? 'zap.bat' : 'zap.sh');
+
+    var cmd = function () {
+      return (options.path ? path.join(options.path, zapScript) : zapScript);
+    };
+
+    var child = (options.os === 'windows' ? spawn(zapScript, args, {cwd: options.path}) : spawn(cmd(), args));
```

Comment:
I tried to spawn this process in Mac OS and Windows. I couldn't find a way to call spawn that works for both systems, so I had to this if. If someone else know a better way, I would love to see it :)